src/meta/processors/admin/AdminClient.cpp

```diff
@@ -603,15 +604,15 @@ folly::Future<Status> AdminClient::createSnapshot(GraphSpaceID spaceId, const st
      * Don't need retry.
      * Because existing checkpoint directories leads to fail again.
      **/
-    getResponse(std::move(allHosts), 0, std::move(req), [] (auto client, auto request) {
+    getResponse({host}, 0, std::move(req), [] (auto client, auto request) {
```

Comment:
Seems you will create snapshot at host one by one?

Reply:
Yes, you are right.